applications/nrf5340_audio/src/modules/Kconfig

```diff
@@ -0,0 +1,121 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "Modules"
+
+config BUTTON_DEBOUNCE_MS
+	int "Button debounce time in ms"
+	default 50
+
+#----------------------------------------------------------------------------#
+menu "Power measurement"
+
+config POWER_MODULE_MIN_MEAS_TIME_MS
+	int "Power measurement interval in milliseconds"
+	default 200
+
+config POWER_MODULE_MEAS_START_ON_BOOT
+	bool "Start power measurements for all rails on boot"
+	default n
+	help
+		This option will automatically start and periodically print
+		the voltage, current consumption, and power usage for the
+		following rails: VBAT, VDD1_CODEC, VDD2_CODEC, and VDD2_NRF
+
+endmenu # Power measurement
+
+#----------------------------------------------------------------------------#
+menu "I2S"
+
+config I2S_ENABLED
```

Comment:
This option does not seem to be used anywhere. Similarly `I2S_TX_ENABLED` and `I2S_RX_ENABLED`. What's their purpose?

Reply:
Will remove